2.ensemble-z-analysis/README.md

```diff
@@ -22,14 +23,30 @@ We compress gene expression data with the following algorithms:
 ## Evaluation Metrics
 
 We will evaluate the solutions across the ensemble population over all z dimensions.
+For each of the populations, we will also track performance of training and testing sets independently.
 
 1. Reconstruction Cost - Measures the binary cross entropy of input data to reconstruction
 2. Training History - For neural network models (ADAGE, Tybalt), save the training progress of each model
    * For Tybalt, the KL Divergence and Reconstruction Loss are saved separately
 3. Correlation of input sample to reconstructed sample - Measure how well certain samples traverse through the bottleneck.
    * Calculate Pearson and Spearman correlations
    * May reveal certain biases in sample reconstruction efficiency across algorithms
+4. Correlation of input gene profile to reconstructed gene profile - Measure how well certain genes are reconstructed through the bottleneck.
+   * Calculate Pearson and Spearman correlations
```

Comment:
Why are you calculating the spearman correlation?

Reply:
Non-normally distributed data should be modeled better with Spearman than Pearson. I can imagine a scenario in which we compare the differential (Spearman - Pearson) across algorithms and observe that the nonlinear neural networks are better when the data is "more" non-normal. Some similar observations were shown in [Cleary et al.](https://doi.org/10.1101/091926)